### DISCUSSION TITLE
UCTRONICS Provides Fix and Offers Replacements for Bad POE Adapters

### DISCUSSION THREAD

**Discussion by Bob-AOL** [gtxaspec/wz_mini_hacks#495] [General]
Through the great t-shooting work of several folks here, along with significant diligence and persistence (tenacity?) directed toward the UCTRONICS Support/R&D group for greater attention to the information being presented to them, a specific problem and solution has been identified with their beleaguered POE adapter -- specifically the new 'Upgraded' version.

Initial communication efforts were slow but steady, primarily due to the language factor occasionally blurring intended specifics and uses of terminology. Once we established a feel for each other, their cooperation increased and the dialogue progressed at a fairly rapid pace.
 
In the absence of their own v3 for testing, (still enroute to them),  they relied entirely on the end-user field testing information being provided in guiding their approach to the issue(s). The group seemed to understand and appreciate the wz_mini_hacks community and anxious to make good on the product accordingly. 
 
Attached are the results of the effort. They welcomed my assistance in crafting the approach attached. They have endorsed its content and encouraged exposure wherever it may be relevent or helpful. I intend to post it in a couple of the other Wyze venues for general digestion as well. 

I hope it is helpful here.

[Realtek Guide.pdf](https://github.com/gtxaspec/wz_mini_hacks/files/10949171/Realtek.Guide.pdf)

**Comment by SomebodySysop** [comment]
Of the companies that I reached out to in my efforts to find a reliable POE adapter for WyzeCam V3, Uctronics was the only one that responded with a true interest in resolving the issues.  I have tested the tool and it does work.  This is a great POE solution for those of us using WyzeCams with Wz_Mini_Hacks.

**Comment by stncttr908** [comment]
I just wanted to offer my thanks to all the diligent folks here for their efforts. I've had excellent luck with my UCTRONICS PoE adapters and Wyze cams and that's entirely thanks to this community.

**Comment by bjs-pdx** [comment]
Unclear on the replacement process.  I had one of the bad adapters and returned it to Amazon.  Is Uctronics offering to replace this?  If so, how do you contact them?

**Reply by SomebodySysop** [reply]
> Unclear on the replacement process. I had one of the bad adapters and returned it to Amazon. Is Uctronics offering to replace this? If so, how do you contact them?

If you returned the adapter to Amazon, you either received a replacement or refund.  If you request a replacement, you may end up with the same problem.  

The refund / replacement options are available in your Amazon.com account, under "Returns and Orders".

If you have a "bad" adapter, I suggest you use the tool to assign it a MAC, which should instantaneously make it a "good" adapter.  As I stated earlier, it worked for me.

**Reply by Bob-AOL** [reply]
The replacement process was clearly stated on the last page of the PDF. That said, the policy is evidently more restricted in scope than I originally understood as I was subsequently informed that it will be on a case-by-case basis only and the offer is limited. Specific details weren't provided, but my sense is there was concern about it getting out of hand if publicized too widely beyond the group here. Fair enough. While never the intention to broadcast the offer any further, I can certainly see them being a little nervous about the possibility. 

They're good folks. Give em a shout and see where it goes. You may well be the only one who does.

**Comment by pjuhl2313** [comment]
Thanks for posting this! Fix worked like a charm. However does anyone have any issue with long distance Ethernet cable and keeping connection with the Uctronics adapter?

**Reply by Bob-AOL** [reply]
If you're still using wi-fi and just using the adapter for its PoE you'll still have the normal dropouts. Otherwise the ethernet connection should be good to around 300 feet at 100Mbps just like any other ethernet connection. I have no issues with several 50 ~ 60 foot runs and achieve upwards of 250kB/s consistently.

**Reply by pjuhl2313** [reply]
Ok. I'm using it as POE at about a 40-50 ft distance. I'm away for work but had a drop out that won't reconnect even with rebooting the specific port. I might just have to try a reset with it when I'm home.

**Comment by donsandrop** [comment]
> [Realtek Guide.pdf](https://github.com/gtxaspec/wz_mini_hacks/files/10949171/Realtek.Guide.pdf)

@Bob-AOL thanks for working through this with the group here and UCTRONICS.
Did they happen to share if they have a macOS version of this tool or if you have a contact we can ask this to?

**Reply by Bob-AOL** [reply]
It's a Windows tool. You'll need emulation or someone with a Windows system to assess the adapter and program the MAC address.  It will run from a thumb drive.
